cypress/e2e/common/cmsSteps.js

```diff
@@ -547,4 +563,30 @@ And('user selects {string} from Source View dropdown', (sourceDropdown) => {
 
 And('user enters {string} in Raw HTML Content under Secondary Contents', (value) => {
     cy.get('table[id*="field-secondary-contents"]').find('textarea[name*="field_raw_html"]').type(value);
+})
+
+When('user clicks on {string} option button',(deleteBtn)=>{
+cy.get("[id*='edit-delete']").click();
+});
```

Comment:
The `deleteBtn` variable isn't used.  This should just be `'user clicks on {string} option button'`.

Reply:
now it is)